### DISCUSSION TITLE
How to close tab without remove the tab

### DISCUSSION THREAD

**Discussion by ogios** [zen-browser/desktop#1734] [Q&A]
Sometimes to save memory, i want to temporarily close some tab and restore it later on.

Like the effect when close and reopen the browser and some tabs go grey, this is much better than save it to the bookmark or find it in history and reopen the tab.

But i don't find this feature in any browser currently

**Comment by kwilcz** [comment]
> But i don't find this feature in any browser currently

This is done in Vivaldi, you can put tabs to sleep on-demand. 
It would be nice to do it on an inactivity period as well just to use less resources.
